Public/index.html

```diff
@@ -1,236 +1,201 @@
 <head>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/Styles.css">
   <title>To-Do List</title>
-  <style>
-    body {
-      font-family: Arial, sans-serif;
-      display: flex;
-      justify-content: center;
-      align-items: center;
-      height: 100vh;
-      margin: 0;
-      background-color: #f4f4f9;
-    }
-
-    .container {
-      text-align: center;
-      max-width: 400px;
-      width: 100%;
-    }
-
-    input {
-      padding: 10px;
-      width: 80%;
-      margin: 10px;
-      border-radius: 4px;
-      border: 1px solid #ccc;
-    }
-
-    button {
-      padding: 10px 15px;
-      background-color: #4caf50;
-      color: white;
-      border: none;
-      border-radius: 4px;
-      cursor: pointer;
-    }
-
-    button:hover {
-      background-color: #45a049;
-    }
-
-    ul {
-      list-style-type: none;
-      padding: 0;
-    }
-
-    li {
-      display: flex;
-      justify-content: space-between;
-      align-items: center;
-      padding: 10px;
-      background-color: #fff;
-      margin: 5px 0;
-      border: 1px solid #ccc;
-      border-radius: 4px;
-      position: relative;
-    }
-
-    li.completed {
-      text-decoration: line-through;
-      background-color: #d3ffd3;
-    }
-
-    .dropdown {
-      position: absolute;
-      top: 0;
-      right: 0;
-      display: none;
-      background-color: #fff;
-      box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2);
-      z-index: 1;
-      border: 1px solid #ccc;
-      border-radius: 4px;
-      padding: 5px;
-      width: 120px;
-    }
-
-    .dropdown-btn {
-      background-color: transparent;
-      border: none;
-      color: #4caf50;
-      cursor: pointer;
-      padding: 5px;
-    }
-
-    .dropdown-btn:hover {
-      color: #45a049;
-    }
-
-    li:hover .dropdown {
-      display: block;
-    }
-
-    .delete-btn {
-      background-color: #e57373;
-      color: white;
-      border: none;
-      cursor: pointer;
-      padding: 5px 10px;
-      border-radius: 4px;
-    }
-
-    .delete-btn:hover {
-      background-color: #f44336;
-    }
-
-    .complete-btn, .incomplete-btn {
-      background-color: #4caf50;
-      color: white;
-      border: none;
-      cursor: pointer;
-      padding: 5px 10px;
-      border-radius: 4px;
-    }
-
-    .complete-btn:hover, .incomplete-btn:hover {
-      background-color: #45a049;
-    }
-
-  </style>
+  
 </head>
 <body>
   <div class="container">
-    <h1>To-Do List</h1>
+    <div class="card">
+      <div class="container_copy">
+    <h1>
+      <img src="images/dooooL.png" alt="To-Do Icon" class="todo-icon">
+      
+      To-Do List!
+     
+      
+    </h1>
+    
+    <div class="row">
     <input type="text" id="task-input" placeholder="Add a new task" />
-    <button id="add-task-btn">Add Task</button>

```

Comment:
Hi Valerie, Your project looks great in terms of design and structure! I have a suggestion that might enhance it even further. When a user adds an item, perhaps consider setting character limits in the input box. This could prevent overlap between the buttons for marking, unmarking, and deleting tasks, helping maintain the clean and polished look of the interface list. Hope this idea is helpful!.